notebooks/icrp107_dataset/icrp107_dataset.ipynb

```diff
@@ -147,6 +147,7 @@
    "source": [
     "year = 365.2422 # number of days in year used for conversion in ICRP-107 \n",
     "                # (see JAERI 1347 & JAEA-Data/Code 2007-021)\n",
+    "amu = 931494.32 # number of keV in one amu (* c^2)\n",
```

Comment:
To make things consistent, let's use the value stated in the AMDC AME2020 paper for this constant (see [Table A](https://iopscience.iop.org/article/10.1088/1674-1137/abddb0/pdf) - 931494.10242).